helm-charts/falcon-sensor/templates/daemonset.yaml

```diff
@@ -133,7 +133,7 @@ spec:
         - name: falconstore-dir
           hostPath:
             path: /var/lib/crowdstrike
-      serviceAccountName: crowdstrike-falcon-sa
+      serviceAccountName: {{ include "falcon-sensor.serviceAccountName" .}}
```

Comment:
This should be `{{ .Values.serviceAccount }}`

Reply:
I'm changing to `{{ .Values.serviceAccount }}` but I was using the serviceAccountName helper, accessing directly to the value we can clean up the code by removing the serviceAccountName Helper in `_helpers.tpl` (I will suggest this modification as well)

Reply:
definition : 
- `helm-charts/falcon-sensor/templates/_helpers.tpl:56:{{- define "falcon-sensor.serviceAccountName" -}}`

places where this helper is used: 
- `helm-charts/falcon-sensor/templates/daemonset.yaml:136:      serviceAccountName: {{ include "falcon-sensor.serviceAccountName" .}}`
- `helm-charts/falcon-sensor/templates/node_scc.yaml:34:- {{include "falcon-sensor.serviceAccountName" .}}`
- `helm-charts/falcon-sensor/templates/container_deployment_webhook.yaml:96:      serviceAccountName: {{ include "falcon-sensor.serviceAccountName" . }}`
- `helm-charts/falcon-sensor/templates/serviceaccount.yaml:4:  name: {{ include "falcon-sensor.serviceAccountName" . }}`
- `helm-charts/falcon-sensor/templates/clusterrolebinding.yaml:20:  name: {{include "falcon-sensor.serviceAccountName" .}}`

Reply:
I have cleaned up everything to match `{{ .Values.serviceAccount }}`